clang/runtime/dpct-rt/include/dpct/device.hpp

```diff
@@ -512,6 +512,10 @@ class device_ext : public sycl::device {
     return get_device_info().get_global_mem_size();
   }
 
+  bool get_virtual_mem_support() const {
```

Comment:
here don't need this help function.  Use has() directory.

Reply:
fixed